media-proxy/include/session-mtl.h

```diff
@@ -0,0 +1,214 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2024 Intel Corporation
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef __SESSION_MTL_H
+#define __SESSION_MTL_H
+
+#include <mtl/st30_api.h>        /* st2110-30 */
+#include <mtl/st40_api.h>        /* st2110-40 */
+#include <mtl/st_pipeline_api.h> /* st2110 pipeline */
+
+#include "session-base.h"
+
+#define MTL_ZERO_COPY
```

Comment:
This flag is added by build script, please remove it from the code.

Reply:
Previous approach was wrong. Macro is defined in media_proxy subdirectory.
When you add ```#include "session-mtl.h"``` inside Unit Tests macro is alwas OFF, while compilation of "media_proxy" submodule happen with macro ON. This lead to smaller object allocation size compared to what library expected, and overall read/write outsied of allocation happen

Reply:
TODO:
Remove ZERO_COPY macro from media_proxy/CMakeLists.txt

Reply:
done